perfkitbenchmarker/benchmark_sets.py

```diff
@@ -32,9 +32,9 @@
         BENCHMARK_LIST: [
             'aerospike', 'cassandra_stress', 'object_storage_service',
             'cluster_boot', 'copy_throughput', 'coremark', 'fio',
-            'hadoop_terasort', 'hpcc', 'iperf', 'mesh_network', 'mongodb',
-            'netperf', 'ping', 'redis', 'speccpu2006',
-            'block_storage_workload', 'sysbench_oltp', 'unixbench']
+            'hadoop_terasort', 'hpcc', 'iperf', 'mesh_network', 'mongodb_ycsb',
```

Comment:
Not that you are adding this to the STANDARD_SET you can remove it from the google_set.  I'd comment there but Github only allows commenting on code changed.